standalone-metastore/metastore-server/pom.xml

```diff
@@ -492,10 +492,13 @@
       </reporting>
     </profile>
     <profile>
-      <id>dist</id>
-    </profile>
-    <profile>
-      <id>docker</id>
+    <id>add-extra-deps</id>
+      <activation>
+        <property>
+          <name>allModules</name>
```

Comment:
Why do we have to add an build option?

Can we include all the relevant dependencies by default?
https://github.com/apache/hive/blob/925df927248c1be11b84df3e8a0c3e43955e133e/standalone-metastore/metastore-server/pom.xml#L499-L512

Reply:
Unfortunately, no - doing so would create a cyclic dependency in the reactor chain. The metastore-rest-catalog module depends on metastore-server, so we'd need to either refactor the structure or introduce a separate packaging module for HMS, similar to how it's done for HS2.

Reply:
I see. Thanks!